src/main/java/com/worksap/fig/lang/Seq.java

```diff
@@ -389,6 +390,32 @@ default int findLastIndex(Predicate<T> condition) {
     }
 
     /**
+     * Create a new seq with size null elements
+     */
+    static <T> Seq<T> newSeq(int size) {
+        return new SeqImpl<>(size);
+    }
```

Comment:
Please follow the original documentation if you can:

```
    /**
     * Constructs an empty list with the specified initial capacity.
     *
     * @param  initialCapacity  the initial capacity of the list
     * @throws IllegalArgumentException if the specified initial capacity
     *         is negative
     */
    public ArrayList(int initialCapacity) {
```

You don't need to spend time to write your own, which is not so better than the original.